crates/anvil/src/eth/api.rs

```diff
@@ -409,10 +409,20 @@ impl EthApi {
         from: &Address,
         request: TypedTransactionRequest,
     ) -> Result<TypedTransaction> {
-        for signer in self.signers.iter() {
-            if signer.accounts().contains(from) {
-                let signature = signer.sign_transaction(request.clone(), from)?;
-                return build_typed_transaction(request, signature)
+        match request {
+            TypedTransactionRequest::Deposit(_) => {
+                return build_typed_transaction(
+                    request,
+                    Signature { r: U256::zero(), s: U256::zero(), v: 0 },
```

Comment:
this should be a constant

Reply:
@mattsse just defined right above, or somewhere else?